artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -1012,6 +1012,33 @@
             </xsd:complexType>
          </xsd:element>
 
+         <xsd:element name="metrics-plugin" maxOccurs="1" minOccurs="0">
```

Comment:
Is the test version of artemis-configuration.xsd updated also, to keep them in sync.